Packs/CommonPlaybooks/Playbooks/Get_File_Sample_-_Generic.yml

```diff
@@ -0,0 +1,377 @@
+id: Get File Sample - Generic
+version: -1
+name: Get File Sample - Generic
+description: Retrieves files from endpoints by the file hash or the file path.
+starttaskid: "0"
+tasks:
+  "0":
+    id: "0"
+    taskid: 2f9db3d3-0254-4d38-8bce-aa2325da9194
+    type: start
+    task:
+      id: 2f9db3d3-0254-4d38-8bce-aa2325da9194
+      version: -1
+      name: ""
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "1"
+      - "2"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 452.5,
+          "y": 50
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "1":
+    id: "1"
+    taskid: 7456030a-675f-4c46-8231-7820167addc6
+    type: playbook
+    task:
+      id: 7456030a-675f-4c46-8231-7820167addc6
+      version: -1
+      name: Get File Sample By Hash - Generic v3
+      description: |-
+        This playbook returns a file sample correlating to a hash in the War Room using the following sub-playbooks:
+        - Get binary file by MD5 hash from Carbon Black telemetry data - VMware Carbon Black EDR v2.
```

Comment:
Is this the name of the sub-playbook?

Reply:
Yes that's its name

Reply:
This is a description coming from another playbook I did not touch. But the correct subplaybooks of it are:
`Get the binary file from Carbon Black by its MD5 hash`
`Get File Sample By Hash - Cylance Protect v2`
`Code42 File Download`